yarn.lock

```diff
@@ -7690,6 +7828,11 @@ jest-docblock@^29.4.3:
   dependencies:
     detect-newline "^3.0.0"
 
+jest-dom@^4.0.0:
```

Comment:
Did you run yarn again after removing the dependency? seems to have persisted in the lock - i'd revert the yarn lock to its original state

Reply:
Good catch.